### PR TITLE
Edit keywords + title

### LINKED ISSUE #4: Title + keywords

### PR DESCRIPTION
**Pull request by lindsayplatt**
Fixes #4. I also added a note about the exact lake counts in the abstract for later. I put "12,000" in the title because it was halfway between what we have for GCM & NLDAS plus was round. Not sure if that is misleading.

Going to ask that @AndyMcAliley review this one to make sure it appropriately encompasses his LSTM work.

### CODE REVIEW

File: in_text/text_data_release.yml [hunk 1]
```diff
@@ -1,6 +1,5 @@
 title: >-
-  Data release: [your title here]
-
+  Daily water column temperature predictions for 12,000 lakes in the Midwest U.S. between 1980-2000 and under future climate scenarios
```
[the remark this answers is not in the document]
Comment (AndyMcAliley) [reply in_text/text_data_release.yml:2]: I'm predicting lakes beyond just the midwest, too. Actually, there's lakes from eastern Montana to Long Island, and from northern Louisiana to southern Ontario and Saskatchewan.
Screen Shot 2023-01-12 at 11 20 32 AM

[the remark this answers is not in the document]
Comment (AndyMcAliley) [reply in_text/text_data_release.yml:2]: If I need to trim down by region, let me know! These are just all the lakes in `8_viz/inout/lakes_summary.feather`.

[the remark this answers is not in the document]
Comment (lindsayplatt) [reply in_text/text_data_release.yml:2]: Wow, so many lakes! OK, so we will definitely need to change the title. Hmmm it feels strange to limit the region but at the same time it is a little strange to have lakes way outside (like in NY/PA when we skip OH essentially). Let's think on this more ...

[the remark this answers is not in the document]
Comment (AndyMcAliley) [reply in_text/text_data_release.yml:2]: Excluding Canada and the northeastern lakes, we're left with 62966 lakes.
bokeh_plot-18

**Comment by lindsayplatt** [comment]
@AndyMcAliley check out title edit

**Review by AndyMcAliley — APPROVED** [review]
Looks good!

**Pull request merged**